docs/ABOUT.md

```diff
@@ -0,0 +1,10 @@
+<!-- this is a stub, copied from factorcode.org. I should really put something better here... and I will! soon -->
```

Comment:
Cool! The thing that I think is great to put at the top of the "about" is _why would I want to learn factor, if I never heard of it before?_ In other words: what kind of problem is it typically used to solve, or how will this make me a better programmer (or better person, for that matter).